messages/endpoint-must-use-cdn.md

```diff
@@ -0,0 +1,29 @@
+# `endpoint` must use CDN
+
+`@prismicio/client` uses either a repository name or a repository endpoint to query content from Prismic.
+
+The repository name or repository endpoint must be provided when creating a `@prismicio/client` like the following:
+
+```typescript
+import * as prismic from "@prismicio/client";
+
+// Using the repository name
+const client = prismic.createClient("example")
+
+// Using the repository endpoint
+const client = prismic.createClient("https://example.cdn.prismic.io/api/v2")
```

Comment:
Going forward, @samlfair recommends we use `example-prismic-repo` when needing a placeholder repository name. The repository is real and contains some example content.